portfolio/src/main/webapp/index.html

```diff
@@ -30,15 +30,23 @@ <h1 id="title">Cindy Hao</h1>
 
         <!-- Basic Personal Information -->
         <div id="about-container" class="container">
-          <h2>About Me</h2>
+          <h2 id="bio-title">About Me</h2>
           <img id="profile-pic" src="/images/cindy.JPEG" />
-          <p>
+          <p id="bio">
               Hi! I'm a student at the University of Pennsylvania. I'm pursuing a Bachelor of Science in Engineering (BSE) in Computer Science and a Master of Science in Engineering (MSE) in Systems Engineering!
               I'm also a former Speech and Debater, bagel enthusiast, loyal friend, baker, and artist (or at least I like to think so). Welcome!
-          </p >
-          <p>
+          </p>
+          <p id="bio-supplement">
             Side note: this page is a little more ~fun~ and is being used largely to showcase my love for cooking. If you want to see some of my work, head on over to <a href="https://www.cindyhao.com"target=”_blank” id="bio-link"> my other website!</a>
           </p>
+          <select id="lang-select">
+            <option value="en" selected>English</option>
```

Comment:
One minor nit: Alphabetize these? You shouldn't need my re-approval after you change this.

Reply:
I alphabetized by value -- do you think it would be better to do it by the name of the language? (can change in next PR if so)

Reply:
ohh oops apparently there's differences between local and remote -- I'm checking what the differences are but do you know what could have caused this?
![image](https://user-images.githubusercontent.com/48307028/84656708-bb7e3180-aed8-11ea-840d-013075eea988.png)

Reply:
just kidding I think I resolved it! (still a little unclear how that happened though)